### PR TITLE
kodi.packages.pvr-iptvsimple: 7.4.2 -> 7.5.1

### PR DESCRIPTION
**Pull request by aanderse**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
I have tested this and it works :+1:

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117430` at 734c5023 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>kodiPackages.inputstream-ffmpegdirect</li>
    <li>kodiPackages.inputstream-rtmp</li>
    <li>kodiPackages.pvr-iptvsimple</li>
  </ul>
</details>

File: pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version = "1.19.4";
+
+  src = fetchFromGitHub {
+    owner = "xbmc";
+    repo = "inputstream.ffmpegdirect";
+    rev = "${version}-${rel}";
+    sha256 = "1ppvs6zybbi73zq1qh8klyhj99byh61c6nijmb1gd5yhg7cywf72";
+  };
+
+  extraBuildInputs = [ bzip2 zlib kodi.ffmpeg ];
```
Comment (SuperSandro2000) [inline pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix:15]: This is that all packages use the same ffmpeg?

Comment (aanderse) [reply pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix:15]: I am sorry but I do not understand your question. Can you please elaborate?

Comment (SuperSandro2000) [reply pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix:15]: Why do we use kodi.ffmpeg instead of ffmpeg?

Comment (aanderse) [reply pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix:15]: `kodi` has its own fork of `ffmpeg` which diverges from upstream in significant ways.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 117430` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>kodiPackages.inputstream-ffmpegdirect</li>
<li>kodiPackages.inputstream-rtmp</li>
<li>kodiPackages.pvr-iptvsimple</li>
</ul>
</details>

**Pull request merged**